src/platform/plugins/shared/discover/server/plugin.ts

```diff
@@ -60,6 +62,13 @@ export class DiscoverServerPlugin
     }
 
     plugins.embeddable.registerEmbeddableFactory(createSearchEmbeddableFactory());
+    plugins.embeddable.registerTransforms(SEARCH_EMBEDDABLE_TYPE, {
+      transformOut: (state) => {
```

Comment:
Make sure to register transforms in public as well so that short link URL state gets transformed.

Reply:
Thanks for the heads up 👍

Reply:
Yes, thank you! Addressed in https://github.com/elastic/kibana/pull/228115/commits/3f6431bcbc4816cb46e1c16450bb238abc2b686d.